codec/src/main/java/io/netty/handler/codec/DefaultHeaders.java

```diff
@@ -67,6 +67,16 @@
     }
 
     /**
+     * Uses the {@link #hashCode()} method to generate the hash code.
+     */
+    public static final class JavaHashCodeGenerator<T> implements HashCodeGenerator<T> {
```

Comment:
The primary reason for this is `CHARSEQUECE_CASE_INSENSITIVE_HASH_CODE_GENERATOR` in `DefaultTextHeaders`.